### PR TITLE
Multi-node logging #1307

### PR DESCRIPTION
**Pull request by briskycat**
https://github.com/EricomSoftwareLtd/SB/issues/1307
Depends on https://github.com/EricomSoftwareLtd/SB/pull/1388

### CODE REVIEW

File: Setup/docker-compose_dev.yml [hunk 1]
```diff
@@ -164,21 +177,17 @@ services:
...
 
   icap-server:
         image: securebrowsing/icap-server:latest
         # user: nobody
         hostname: icap-server
         environment:
             - NODE_ENV=production
-            - SHIELD_VER=8.0.0.latest
-            - "SHIELD_IP=IP_ADDRESS"
+            - SHIELD_VER=Dev:Build_210
```
Comment (BenyH) [inline Setup/docker-compose_dev.yml]: Please replace with 8.0.0.latest

File: Setup/docker-compose_dev.yml [hunk 2]
```diff
@@ -164,21 +177,17 @@ services:
...
+            - "SHIELD_IP=192.168.56.110"
```
Comment (BenyH) [inline Setup/docker-compose_dev.yml]: Please replace with IP_ADDRESS

File: Setup/docker-compose_dev.yml [hunk 3]
```diff
@@ -313,13 +310,9 @@ services:
...
               cpus: "1"
-        logging:
-          driver: syslog
-          options:
-            syslog-address: udp://${SYS_LOG_HOST}:5014
-            tag: "proxy-{{.ID}}"
+
   ext-proxy:
-        image: securebrowsing/extproxy:latest
+        image: securebrowsing/extproxy:170724-21.57
```
Comment (BenyH) [inline Setup/docker-compose_dev.yml]: Please replace with latest

File: Setup/docker-compose_dev.yml [hunk 4]
```diff
@@ -374,8 +363,9 @@ services:
         networks:
           - shield
         environment:
-            - "SHIELD_IP=IP_ADDRESS"
+            - "SHIELD_IP=192.168.56.110"
```
Comment (BenyH) [inline Setup/docker-compose_dev.yml]: Please replace with IP_ADDRESS

**Comment by BenyH** [comment]
Hi Andrew,
Could you summarize what we are doing here?
If i understand, we are adding a new component logspout that forward syslog from local host to elk (inside shield network)

File: Setup/docker-compose_dev.yml (hunk 4 above)
Comment (briskycat) [reply Setup/docker-compose_dev.yml]: Reverted everything in https://github.com/EricomSoftwareLtd/Shield/pull/47/commits/8398ad3972036beca9073f52f7c444e2fd7a5184
